de.fu_berlin.inf.dpp.ui/build.gradle

```diff
@@ -0,0 +1,63 @@
+buildscript {
+  repositories {
+      maven { url 'https://plugins.gradle.org/m2/' }
+  }
+  dependencies {
+    classpath "com.diffplug.gradle:goomph:3.17.0"
+  }
+}
+
+sourceCompatibility = '1.6'
```

Comment:
If -source is 1.6, the value of -target is 1.7

https://docs.oracle.com/javase/7/docs/technotes/tools/windows/javac.html#crosscomp-options

Please define the target compability.

Reply:
I tried to compile with sourceCompatibility = '1.6' and targetCompatiblity = '1.7', but the javac of jdk 1.6 does not know v. 1.7. Therefore I get the error message `javac: invalid target release: 1.7`.